README.md

```diff
@@ -35,6 +35,27 @@ You can use the following command if you want to deploy `kafka` and related serv
 $ docker-compose -f docker-compose.yml -f docker-compose.kafka.yml up -d 
 ```
 
+Setup pgadmin to view moodle database
```

Comment:
This isn't strictly necessary for "deploying a local environment" and more of an optional tool you an use / setup. Let's perhaps instead make this it's own section as `### Setup pgadmin to view moodle database`